src/main.cc

```diff
@@ -401,7 +401,9 @@ std::vector<Option> options_
        "The port for the server to listen on for HTTP requests."},
       {OPTION_REUSE_HTTP_PORT, "reuse-http-port", Option::ArgBool,
        "Allow multiple servers to listen on the same HTTP port when every "
-       "server has this option set."},
+       "server has this option set. If you plan to use this option as a way to "
+       "load-balance between different triton servers, the same model "
```

Comment:
Here and below:
- Triton should be capitalized.
- Load balance should not have a dash.

Reply:
Thanks for the catch! Updated.